fuzztest/grammars/JSON.g4

```diff
@@ -16,7 +16,7 @@
 //
 // With some simplifications:
 // - Restricted character set (e.g., no unicode chars).
-// - No escape sequences (e.g., \n, \t, \uff01, etc.)
+// - No unicode escape sequences (e.g., \uff01)
```

Comment:
After at first adding the unicode escape sequences, I had to remove them again because nlohmann/json actually validates that the generated codepoints are "proper":
https://github.com/nlohmann/json/blob/a259ecc51e1951e12f757ce17db958e9881e9c6c/include/nlohmann/detail/input/lexer.hpp#L359-L381